AspNetCoreVerifiableCredentials/README.md

```diff
@@ -94,7 +94,7 @@ Make sure you copy the `ClientId`, `ClientSecret` and `TenantTd` you copied when
 
 1. Open a command prompt and run the following command:
 ```Powershell
-dotnet build "asp-net-core-api-idtokenhint.csproj" -c Debug -o .\bin\Debug\netcoreapp3.1
+dotnet build "AspNetCoreVerifiableCredentials.csproj" -c Debug -o .\bin\Debug\net5
```

Comment:
I think we still support code from .net3.1 @kalyankrishna1  might have an opinion on this as well

Reply:
The project is a .NET 5, so I updated this to match. If we revert back to 3.1, then maybe the proj as well? What do you recommend?